src/ioctls/device.rs

```diff
@@ -292,15 +289,7 @@ mod tests {
         set_supported_nr_irqs(&device_fd, 128);
 
         // Following attribute works with VGIC, they should be accepted.
```

Comment:
Let's also update this comment (something like "initialize valid vgic"?)

Reply:
Updated :)